copulas/multivariate/gaussian.py

```diff
@@ -100,42 +80,88 @@ def fit(self, X):
         """
         LOGGER.info('Fitting %s', self)
 
+        # Validate the input data
+        X = self._validate_input(X)
+        columns, univariates = self._fit_individual_columns(X)
```

Comment:
for simplicity we can probably just call _fit_columns

Reply:
Addressed in [6d3dc4c](https://github.com/sdv-dev/Copulas/pull/436/commits/6d3dc4c8f467c25fdfcabb9ec422195e88c3dba2)